src/main/java/com/driving/school/api/StudentApi.java

```diff
@@ -0,0 +1,36 @@
+package com.driving.school.api;
+
+import com.driving.school.model.StudentModel;
+import com.driving.school.service.StudentService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequiredArgsConstructor
+@CrossOrigin(origins = "http://localhost:4200")
+@RequestMapping(value = "/students")
```

Comment:
`@RequestMapping("/students")` - nie musi być tego value, ale nie ma to większego znaczenia jak dla mnie